packages/dataviews/src/dataviews-layouts/grid/style.scss

```diff
@@ -30,12 +30,14 @@
 			.dataviews-view-grid__fields .dataviews-view-grid__field .dataviews-view-grid__field-value {
 				color: $gray-900;
 			}
-
-			.dataviews-view-grid__media::after {
-				background-color: rgba(var(--wp-admin-theme-color--rgb), 0.08);
-				box-shadow: inset 0 0 0 $border-width var(--wp-admin-theme-color);
-			}
 		}
+		&.is-selected .dataviews-view-grid__media::after,
```

Comment:
It works for me, but I welcome feedback from design: in the grid view, do we need to distinguish from a "selected" state and from a "focused" state? (I presume this would be fine to tweak as a follow-up if necessary)

| Selected | Focused |
| --- | --- |
| <img width="1202" alt="Screenshot 2024-12-10 at 14 27 05" src="https://github.com/user-attachments/assets/1587958a-ad30-4368-858d-a79edee79402"> | <img width="1202" alt="Screenshot 2024-12-10 at 14 27 01" src="https://github.com/user-attachments/assets/2a39ae08-d785-49f1-8bf8-f4b52d34d433"> |

cc @jameskoster

Reply:
Thanks for the ping. The focus treatment should utilise the `var(--wp-admin-border-width-focus)` (`1.5px`) stroke width. The stroke for the selected item should be only `1px`.